Backend/src/database/models/project.ts

```diff
@@ -1,63 +1,51 @@
 import {Document, model, Schema} from "mongoose";
 
-export enum visibilityStatus {
-    "PRIVATE",
-    "PUBLIC",
-    "BY_INSTITUTION",
+export enum ProjectStatus {
+    "UPLOAD_PENDING",
+    "UPLOAD_COMPLETE",
+    "PROCESSING_PENDING",
+    "ABORTED",
+    "DONE"
 }
+
 export interface IProject extends Document {
-   title:string,
-   description:string,
-   adminIds: Array<Schema.Types.ObjectId>,
-   invitedMemberIds: Array<Schema.Types.ObjectId>,
-   memberIds: Array<Schema.Types.ObjectId>,
-   visibility: string,
-   institutionId: Schema.Types.ObjectId,
+    owner: Schema.Types.ObjectId;
+
+    // file
+    uploadId: string;
+    location: string;
+    fileName: string;
+    fileSize: number; // (of bytes)
+    uploadDate: Date;
+
+    // team
+    status: string;
+    resultName: string;
+    resultSize: number;
+
+    teamId: Schema.Types.ObjectId
 }
 
 const projectSchema = new Schema<IProject>({
-    title: {
-        type: String,
-        required: true
-    },
-
-    description: {
-        type: String, 
-        require: true
-    },
+    owner: {type: Schema.Types.ObjectId, require: true},
 
-    adminIds: [{
-        type: Schema.Types.ObjectId, 
-        ref: 'User',
-        require: true
-    }],
+    // file
+    uploadId: {type: String, require: false},
+    fileName: {type: String, require: false},
+    location: {type: String, require: false},
+    fileSize: {type: Schema.Types.Number, require: false, default: -1},
+    uploadDate: {type: Schema.Types.Date, require: true},
 
-    invitedMemberIds: [{
-        type: Schema.Types.ObjectId, 
-        ref: 'User',
-        require: false
-    }],
+    // team
```

Comment:
```suggestion
    // project
```
Still stores information about the project status